engine/src/main/java/org/terasology/engine/recording/EventSystemReplayImpl.java

```diff
@@ -276,7 +276,7 @@ private EntityRef getEntityRef(RecordedEvent recordedEvent) {
     @Override
     public void registerEvent(ResourceUrn uri, Class<? extends Event> eventType) {
         eventIdMap.put(uri, eventType);
-        logger.debug("Registering event {}", eventType.getSimpleName());
+        logger.atDebug().addArgument(() -> eventType.getSimpleName()).log("Registering event {}");
```

Comment:
O(1) method call only called once when registering an event, so impact considered neglectable
```suggestion
        logger.debug("Registering event {}", eventType.getSimpleName()); //NOPMD
```